### DISCUSSION TITLE
cannot set the area under the file-tree to be transparent

### DISCUSSION THREAD

**Discussion by X-Yunocchi-X** [nvim-tree/nvim-tree.lua#2111] [Q&A] [answered]
![neovide](https://user-images.githubusercontent.com/106807558/230535434-b39edaa0-5f98-4077-a97c-7c86b682d095.png)

i wish the area under the file-tree would be transparent too, but i failed. i use nvim-transparent plugins to do that, so what's the name of the opaque area, i didn't find it in the document(:help nvim-tree). help me!!

**Comment by gegoune** [comment]
See what `NvimTreeNormal` group looks like.

Why do you need a plugin for making it transparent?

**Reply by kevinrobayna** [reply]
I'm also on the same boat as above, I'm using folke/tokyonight.nvim which provides support for transparent background and it works for the header part of nvim-tree but not the remaining 😅 

![image](https://user-images.githubusercontent.com/5471469/232709521-aa215c08-d670-49f2-a9d7-58754e393e68.png)

**Reply by kevinrobayna** [reply, marked as answer]
actually just realised that it was my fault for not checking [folke's documentation](https://github.com/folke/tokyonight.nvim/blob/main/lua/tokyonight/config.lua#L18) 😅 

You're right most of the times you shouldn't need another plugin messing around the theme itself should support it.
